extension-manifest-v3/src/utils/engines.js

```diff
@@ -22,6 +22,17 @@ import { registerDatabase } from '/utils/indexeddb.js';
 
 export const CUSTOM_ENGINE = 'custom-filters';
 
+const checkUserAgent = (pattern) => navigator.userAgent.indexOf(pattern) !== -1;
+
+const ENV = new Map([
```

Comment:
This can be then inlined rather than global in the module scope.

Reply:
it is used after `deserialise` and for `updateFromDiff`